src/debug.cpp

```diff
@@ -19,6 +19,8 @@ limitations under the License.
 #include "libcellml/generator.h"
 #include "libcellml/variable.h"
 
+#undef NAN
```

Comment:
is this needed?

Reply:
Yes, one of the system headers defines `NAN` as a macro while we use it in one a class enum, so this causes conflicts. This has been discussed somewhere, but can't find the thread in less than a minute, so giving up. :)